packages/primitives/src/Field/Field.tsx

```diff
@@ -5,22 +5,24 @@ import { Input } from '../Input'
 import { Text } from '../Text'
 
 interface FieldProps {
+  children?: React.ReactNode
+  defaultValue?: string
   label: string
   name: string
-  defaultValue?: string
-  children?: React.ReactNode
+  placeholder?: string
+  type?: string
 }
 
 export const Field = React.forwardRef(
   (
-    { label, name, defaultValue, children }: FieldProps,
+    { children, defaultValue, label, name, placeholder, type }: FieldProps,
```

Comment:
```suggestion
    { children, defaultValue, label, name, ...props }: FieldProps,
```

Reply:
Sounds good to me, incorrectly assumed you were want to limit the props passed in